ui/ChaosKit/OutlinedButton.qml

```diff
@@ -0,0 +1,35 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Controls.impl 2.12
```

Comment:
I think you don't need this import if you remove the override for `contentItem`.